src/Providers/AnnualReportProvider.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Ageras\Sherlock\Providers;
+
+use GuzzleHttp\Client;
+use Ageras\Sherlock\Models\AnnualReport;
+
+class AnnualReportProvider implements IAnnualReportProvider
```

Comment:
This name has to be more specific. `AnnualReportProvider` is a common name. It should be named `CvrAnnualReportProvider` or something similar